In [1]:
import pandas as pd
import numpy as np
from scipy import stats
from scipy.stats import norm

# Q.1

In [2]:
z_score = (50 - 45) / 8
z_score

0.625

In [7]:
1 - stats.norm.cdf(abs(0.625))

0.26598552904870054

In [6]:
p_value = stats.norm.cdf(abs(z_score)) 
p_value

0.7340144709512995

In [8]:
1 - 0.7340144709512995

0.26598552904870054

# Q.2

In [9]:
z_score = (44 - 38) / 6
z_score

1.0

In [12]:
100 - 84.13

15.870000000000005

15.87% => 64 / 400

In [13]:
z_score = (38 - 38) / 6
z_score

0.0

In [14]:
84.13 - 50

34.129999999999995

34.13% => 137 / 400

Employees between age 38 and 44 are older than the employees older than 44. Hence the given statement is false.  

In [15]:
z_score = (30 - 38) / 6
z_score

-1.3333333333333333

-1.33 => 9.15%
9.15% =>  36 / 400

A training program for employees under the age of 30 at the center would be expected to attract about 36 employees. Given statement is true.

# Q.3

X1 ~ N(μ1, σ1^2) 

X2 ~ N(μ2, σ2^2)

X1 + X2 ~ N(μ1 + μ2, σ1^2 + σ2^2)

2X1 ~ N(2μ1, 2σ1^2)

(2X1 - X1 + X2) ~ N(2μ1 - μ1 + μ2, 2σ1^2 - σ1^2 + σ2^2)

=> They have same probability distribution.

# Q.4

In [2]:
stats.norm.interval(0.99,100,20)

(48.48341392902199, 151.516586070978)

=> a = 48.5, b = 151.5

# Q.5

In [5]:
mean = 225 + 315
print ("Mean profit is Rs.", mean , "million")

Mean profit is Rs. 540 million


In [9]:
stdd = np.sqrt(9 + 16)
print ("Standadrd deviation is Rs.", stdd*45, "million")

Standadrd deviation is Rs. 225.0 million


In [10]:
print("Range is Rs.", (stats.norm.interval(0.95,540,225)), "in million")

Range is Rs. (99.00810347848784, 980.9918965215122) in million


In [11]:
# X = μ + Zσ
# Z value for 5th percentile is -1.645 

 
X= 540 + (-1.645) * (225)
print ("5th percentile of profit is Rs.", np.round(X,), "million")

5th percentile of profit is Rs. 170.0 million


In [14]:
stats.norm.cdf(0,5,3)

0.0477903522728147

In [15]:
stats.norm.cdf(0,7,4)

0.040059156863817086